airbyte-integrations/connectors/source-declarative-manifest/pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.78.6"
+name = "source-declarative-manifest"
+description = "Base source implementation for low-code sources."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/low-code"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+[[tool.poetry.packages]]
+include = "source_declarative_manifest"
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "0.78.6"
```

Comment:
This (and the lockfile) gets bumped by the `bump cdk dependency of source-declarative-manifest` step